cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_adapters.go

```diff
@@ -184,3 +185,126 @@ func (s *Server) getInstalledPackageSummary(pkgInstall *packagingv1alpha1.Packag
 	}
 	return installedPackageSummary, nil
 }
+
+func (s *Server) getInstalledPackageDetail(pkgInstall *packagingv1alpha1.PackageInstall, pkgMetadata *datapackagingv1alpha1.PackageMetadata, pkgVersionsMap map[string][]pkgSemver, app *kappctrlv1alpha1.App, valuesApplied, cluster string) (*corev1.InstalledPackageDetail, error) {
+	// get the versions associated with the package
+	versions := pkgVersionsMap[pkgMetadata.Name]
+	if len(versions) == 0 {
+		return nil, fmt.Errorf("no package versions for the package %q", pkgMetadata.Name)
+	}
+
+	deployStdout := ""
+	deployStderr := ""
+	fetchStdout := ""
+	fetchStderr := ""
+	inspectStdout := ""
+	inspectStderr := ""
+
+	if app.Status.Deploy != nil {
```

Comment:
I was wondering why we were needing the kapp `app` resource.

Reply:
I haven't been able to find a better place to look up this status information. Not sure if it is what we really want to display, but, at least it's is something for this initial version :sweat_smile: